andela/ah-jumanji-frontend
# #163519769 Add authors profile to article view

#### What does this PR do?
Add follow button to the view article page next to the username

#### Description of Task to be completed?
- add the profile component to the view
- add the follow button to the view

#### How should this be manually tested?
1. Run the following commands in Terminal:
```
git clone https://github.com/andela/ah-jumanji-frontend.git
git checkout ft-follow_component_on_view-163519769
touch .env
vi .env
```
2. Edit the`.env` file to add:
```
FIREBASE_API_KEY=AIzaSyDLqlYrfTIkjXb01oA_9svo107jkV-YzAg
FIREBASE_AUTH_DOMAIN=ah-jumanji.firebaseapp.com
SOCIAL_AUTH_API_URL=https://ah-jumanji-staging.herokuapp.com/api/users/social/auth
ARTICLES_API_URL=https://ah-jumanji-staging.herokuapp.com/api/articles/
BASE_URL=https://ah-jumanji-staging.herokuapp.com
```
3. Continue in `Terminal`:
```
npm install
npm start
```
4. Login.
5. click on an article to view
`if the article belongs to you, you can see the edit button on the view page`

#### Any background context you want to provide?
It is a great UX for users to see the details of the author whose article they are reading.

#### What are the relevant pivotal tracker stories?
[Add a follow button to the view article page next to the username #163519769](https://www.pivotaltracker.com/n/projects/2225604/stories/163519769)

#### Screenshots (if appropriate)
<img width="1001" alt="screenshot 2019-02-01 at 12 51 48" src="https://user-images.githubusercontent.com/41139653/52115653-331f2980-2620-11e9-8b9a-6781f9020ab5.png">


#### Questions:

#### Checklist:
